locales/fr-FR.json

```diff
@@ -27,15 +27,15 @@
     "moved_title": "a indiqué que son nouveau compte est désormais :",
     "muted_users": "Utilisateur·ice·s masqué·e·s",
     "muting": "Masqué·e",
-    "mutuals": "@:account.following",
+    "mutuals": "Abonné·e·s",
     "pinned": "Épinglés",
     "posts": "Messages",
     "posts_count": "{0} Messages",
     "profile_description": "En-tête du profil de {0}",
     "profile_unavailable": "Profil non accessible",
-    "unblock": "Unblock",
+    "unblock": "Débloquer",
     "unfollow": "Ne plus suivre",
-    "unmute": "Unmute"
+    "unmute": "Démasquer"
```

Comment:
On Twitter, the word used is "Réafficher" (see https://help.twitter.com/fr/using-twitter/twitter-mute)
![image](https://user-images.githubusercontent.com/69633530/212346997-d954dffd-d86d-4f5f-adf4-8f9b9491e679.png)
Wdyt?

Reply:
Deal!